pkgs/by-name/ti/timr-tui/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  alsa-lib-with-plugins,
+  alsa-plugins,
+  pipewire,
+  writableTmpDirAsHomeHook,
+  enableSound ? false,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "timr-tui";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "sectore";
+    repo = "timr-tui";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-s2FnMwDq4tYBaWaT9y1GLbVGFs7zSnOmjcF5leO12JE=";
+  };
+
+  cargoHash = "sha256-9yd348QGjFxt+QmEBuYzd612mFm/PyETrZy4z5wW+nI=";
+
+  # Enable upstream "sound" feature when requested
+  buildFeatures = lib.optionals enableSound [ "sound" ];
+
+  nativeBuildInputs = lib.optionals (enableSound && stdenv.isLinux) [ pkg-config ];
+
+  # Runtime/FFI deps for the sound feature (Linux)
+  buildInputs = lib.optionals (enableSound && stdenv.isLinux) [
+    (alsa-lib-with-plugins.override {
+      plugins = [
+        alsa-plugins
+        pipewire
+      ];
+    })
+  ];
+
+  nativeCheckInputs = [
+    writableTmpDirAsHomeHook
+  ];
+
+  meta = {
+    description = "TUI to organize your time: Pomodoro, Countdown, Timer.";
```

```suggestion
    description = "TUI to organize your time: Pomodoro, Countdown, Timer";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description